mahjong/record/utils/value/tile.py

```diff
@@ -1,14 +1,21 @@
 from ..constant import TENHOU_TILE_CATEGORY, SUIT_ORDER
-from mahjong.tile.definition import Tile
+from mahjong.tile.definition import Tile,AkaTile
 
 
 def tile_from_tenhou(index):
     """
     tenhou index is numbered as [1m,1m,1m,1m,2m,2m,...,9m,1p,...,9p,1s,...,9s,1z,...,7z,7z,7z,7z]
     numbered first 5m,5s,5p is considered as aka dora.
     """
-    color, number, _ = TENHOU_TILE_CATEGORY.category(index)
-    return Tile(number + 1, SUIT_ORDER[color])
+    if index == 16:
```

Comment:
the number `16,52,88` is counter-intutive at first. consider using something like `TENHOU_TILE_CATEGORY.index(('m', 5, 0))` , like below `tile_to_tenhou_range()` does.